Release version 1

ilario-pierbattista opened the pull request:
Release 1 aims to provide full compatibility with newer versions of PHP (>= 8.1) and PHPUnit (>= 10).

- [RC1](https://github.com/giorgiosironi/eris/releases/tag/1.0.0-rc1)

Pixelshaped commented:
Hi there! What do you think is needed to make progress on that PR? It forces some of my bundle to have `"minimum-stability":"dev"` because I need the RC2 to have the attributes (which I contributed). Not urgent though :)

Pixelshaped commented:
Hi @ilario-pierbattista! Please tell me how we can move towards a release. How can I help? :)

ilario-pierbattista commented:
Hey @Pixelshaped just merged this to master and released [1.0.0](https://github.com/giorgiosironi/eris/releases/tag/1.0.0).

Right before merging I've changed the CI test matrix to add PHP 8.4 and fixed some deprecations.

Hope everything is ok with 1.0.0.

Thank you for your help.
